backend/src/main/java/com/ody/mate/service/MateService.java

```diff
@@ -19,15 +16,17 @@ public class MateService {
     private final MateRepository mateRepository;
 
     @Transactional
-    public Mate save(MateSaveRequest mateSaveRequest, Meeting meeting, Member member) {
-        if (mateRepository.existsByMeetingIdAndNicknameNickname(meeting.getId(), mateSaveRequest.nickname())) {
+    public Mate save(Mate mate) {
+        if (mateRepository.existsByMeetingIdAndNicknameNickname(
+                mate.getMeeting().getId(),
+                mate.getNickname())
+        ) {
             throw new OdyBadRequestException("모임 내 같은 닉네임이 존재합니다.");
         }
-
-        if (mateRepository.existsByMeetingIdAndMemberId(meeting.getId(), member.getId())) {
+        if (mateRepository.existsByMeetingIdAndMemberId(mate.getMeeting().getId(), mate.getMember().getId())) {
```

Comment:
[제안]

id보다 meeting이나 member로만 조회하는 건 어떨까요? HQL내에서 자동으로 id(식별자)로 탐색을 해서 성능상 손실은 없습니다.
```suggestion
        if (mateRepository.existsByMeetingAndMember(mate.getMeeting(). mate.getMember())) {
```

Reply:
처음 알게됬는데 JPQL을 보니 객체를 파라미터로 줬을 때 자동으로 id 식별자로 탐색하는 거 같네요 👍🏻
하지만 Repository DB 저장소의 특성에 맞게 데이터에 접근하는 방식을 좀 더 명확히 표현해주는게 어떨까요 ??